Mod Menu Helper/assets/modmenu/lang/ko_kr.json

```diff
@@ -0,0 +1,55 @@
+{
+    "modmenu.dropInfo.line1": "FO 모드는 §9✎§r 또는 §8✎§r, 대체제는 §c*§r로 표시됩니다",
+    "modmenu.dropInfo.line2": "다른 모드는 지원되지 않습니다",
+    "modmenu.summaryTranslation.slight-gui-modifications": "애니메이션, 메뉴의 모드팩 버전\n§9✎§8 애니메이션, 색, FPS 인디케이터, ...",
+    "modmenu.summaryTranslation.advancementinfo": "커지고 스마트해진 도전과제 창\n§9✎§8 색, 정보 칸 크기, ...",
+    "modmenu.summaryTranslation.advancements-enlarger": "§c*§r 커진 도전과제 창\n§8✎ (옵션 없음)",
+    "modmenu.summaryTranslation.animatica": "Optifine의 애니메이션 텍스쳐\n§8✎ (고급 옵션만 있음)",
+    "modmenu.summaryTranslation.antighost": "/ghost 명령어로 Ghost 블록 제거\n§9✎§8 키바인드",
+    "modmenu.summaryTranslation.betterbeds": "§c*§r 빨라진 침대 렌더\n§8✎ (옵션 없음)",
+    "modmenu.summaryTranslation.bettermounthud": "말을 타고있을 때 허기와 경험치 표시\n§8✎ (옵션 없음)",
+    "modmenu.summaryTranslation.bobby": "서버에서 계속 청크가 보이도록 로드\n§9✎§8 블록 엔티티, 청크 어둡게, 캐시 저장, ...",
+    "modmenu.summaryTranslation.borderlessmining": "전체 화면에서 화면 이동 (opt-in)\n§9✎§8 모드 토글, macOS 지원, 커스텀 크기, ...",
```

Comment:
"Opt-in" means "can be enabled manually if wanted". Perhaps there is also Korean phrase for that?

Reply:
I couldn't remember it for a while during the translation, so I passed it over, but I forgot. I fixed it!